src/modules/stores/innovation/innovation-record/innovation-record-schema/innovation-record-schema.store.ts

```diff
@@ -0,0 +1,161 @@
+import { Injectable } from '@angular/core';
+import { InnovationRecordSchemaModel, IrSchemaTranslatorMapType } from './innovation-record-schema.models';
+import { InnovationRecordSchemaService } from './innovation-record-schema.service';
+import { Store } from '@modules/stores/store.class';
+import { ContextStore } from '@modules/stores/context/context.store';
+import { SectionStepsList } from '@modules/shared/pages/innovation/sections/section-summary.component';
+import { WizardIRV3EngineModel } from '@modules/shared/forms/engine/models/wizard-engine-irv3-schema.model';
+import { FormEngineModelV3 } from '@modules/shared/forms/engine/models/form-engine.models';
+import { irSchemaTranslationsMap } from '../202405/ir-v3-schema-translation.helper';
+import { stepsLabels } from '../202304/section-2-2-evidences.config';
```

Comment:
This could be copied to the `202405` version.

Reply:
Saved this, and will be left to do in the future